packages/quantic/force-app/main/default/lwc/quanticNumericFacet/__tests__/quanticNumericFacet.test.js

```diff
@@ -119,85 +175,210 @@ function mockSuccessfulHeadlessInitialization() {
+        exampleEngine.id,
+        expectedFacetType,
+        expect.objectContaining({
+          label: defaultOptions.label,
+          facetId: defaultOptions.facetId,
+          element: element,
+        })
       );
+    });
 
-      expect(generateFacetDependencyConditions).toHaveBeenCalledTimes(2);
-      expect(generateFacetDependencyConditions).toHaveBeenCalledWith({
-        [exampleFacetDependency.parentFacetId]:
-          exampleFacetDependency.expectedValue,
+    it('should subscribe to the headless numeric facet state and search status state changes', async () => {
+      createTestComponent();
+      await flushPromises();
+
+      expect(functionsMocks.facetStateSubscriber).toHaveBeenCalledTimes(1);
+      expect(functionsMocks.searchStatusStateSubscriber).toHaveBeenCalledTimes(
+        1
+      );
+    });
+
+    describe('the facet conditions manager', () => {
+      it('should build the controller when the dependsOn property is set', async () => {
+        const exampleFacetDependency = {
+          parentFacetId: 'filetype',
+          expectedValue: 'txt',
+        };
+        createTestComponent({
+          ...defaultOptions,
+          dependsOn: exampleFacetDependency,
+          withInput: true,
+        });
+        await flushPromises();
+
+        expect(functionsMocks.buildNumericFacet).toHaveBeenCalledTimes(1);
+        expect(functionsMocks.buildNumericFilter).toHaveBeenCalledTimes(1);
+        expect(
+          functionsMocks.buildFacetConditionsManager
+        ).toHaveBeenCalledTimes(2);
+        expect(functionsMocks.buildFacetConditionsManager).toHaveBeenCalledWith(
+          exampleEngine,
+          {
+            facetId: exampleFacetId,
+          }
+        );
+        expect(functionsMocks.buildFacetConditionsManager).toHaveBeenCalledWith(
+          exampleEngine,
+          {
+            facetId: exampleFilterId,
+          }
+        );
+
+        expect(generateFacetDependencyConditions).toHaveBeenCalledTimes(2);
+        expect(generateFacetDependencyConditions).toHaveBeenCalledWith({
+          [exampleFacetDependency.parentFacetId]:
+            exampleFacetDependency.expectedValue,
+        });
+      });
+
+      it('should not build the controller when the dependsOn property is not set', async () => {
+        createTestComponent({
+          ...defaultOptions,
+          withInput: true,
+        });
+        await flushPromises();
+
+        expect(functionsMocks.buildNumericFacet).toHaveBeenCalledTimes(1);
+        expect(
+          functionsMocks.buildFacetConditionsManager
+        ).toHaveBeenCalledTimes(0);
+        expect(generateFacetDependencyConditions).toHaveBeenCalledTimes(0);
       });
     });
 
-    it('should not build the controller when the dependsOn property is not set', async () => {
-      createTestComponent({
-        ...defaultOptions,
-        withInput: true,
+    describe('the numeric filter controller', () => {
```

Comment:
Isn't it more of a `describe('the withInput property')` instead?

`test('when true, it should initialize the numeric filter controller and subscribe to its state changes')`
`test('when false, it should not initialize the numeric filter controller and not subscribe to its state changes')`

Jest has both `it` and `test`, as an alias for the same thing, I think it's for when the test description doesn't work well starting with `it`.